website/content/docs/plugins/creation/index.mdx

```diff
@@ -214,87 +214,39 @@ Here's what you need to create releases using GitHub Actions:
    to install a plugin using `packer init`, see the
    [init docs](/packer/docs/commands/init).
 
-## Registering Plugin Documentation
+## Plugin Integration Registration
 
-~> Note: Registering a remote plugin's plugin documentation requires the use of [Packer's plugin docs configuration](https://github.com/hashicorp/packer-plugin-scaffolding/tree/main/docs).
+~> Note: Registering a remote as an integration requires the use of [Packer's plugin docs configuration](https://github.com/hashicorp/packer-plugin-scaffolding/tree/main/docs).
 
-`packer init` allows users to require and install remote Packer plugins, those not bundled with Packer core, that have been published to GitHub automatically.
-To help with the discovery of remote Packer plugins on GitHub, plugins maintainers can choose to register plugin documentation for each component directly on the [Packer Documentation Page](/packer/docs).
+`packer init` allows users to require and install remote Packer plugins published to GitHub, those not bundled with Packer core, automatically.
+To help with the discovery of remote Packer plugins, plugins maintainers can choose to register their plugin as a [Packer Integration](/packer/integrations).
 
-The registration process requires the creation of a `docs.zip` file archive containing the `.mdx` files for each of the plugin components in the remote plugin's repository. A working example can be seen at the [packer-plugin-docker repository](https://github.com/hashicorp/packer-plugin-docker/releases/latest).
+The registration process requires changes to your plugin repository for configuring the Packer integration pipeline and plugin documentation for rendering 
```

Comment:
Should we say that the registration requires changes here? This is the case for plugin authors that adopted the scaffolding before this change, but once we've updated the structure, this won't apply, as the new structure will be in place.